Sources/Internal/ItemViewReuseManager.swift

```diff
@@ -25,6 +25,7 @@ final class ItemViewReuseManager {
 
   func viewsForVisibleItems(
     _ visibleItems: Set<VisibleItem>,
+    recycleOldViews: Bool,
```

Comment:
Something more descriptive?
```suggestion
    recycleOffscreenViews: Bool,
```

Reply:
Nice, I like that

Reply:
They might not be offscreen, so going with `recycleUnusedViews` (imagine we just switch from using `DayViewA` to `DayViewB` for a visible day, it's not really offscreen in terms of being within the visible bounds of the scroll view)